turbo.json

```diff
@@ -25,6 +25,7 @@
       "dependsOn": [
         "lint"
       ]
-    }
+    },
+    "test": {}
```

Comment:
No, remove this change. Turbo doesn't run the tests; only Vitest does.

Reply:
This runs all test steps in the subpackages though, so if we ever wanted to add tests to the backend it would run both

Reply:
We also get caching via doing it through Turbo

Reply:
Hmm, honestly I think I'd prefer opting out of using turborepo for this; I'd rather that all the tests ran all the time so that way we could always know that everything is working properly. Until we have enough tests for the caching to matter, I think it would be easier to do more work, but be more consistent.